src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java

```diff
@@ -384,26 +385,37 @@ else if (posY > maxY - 100)
 			
 			g2.setStroke(new BasicStroke(2));
 
-			// displays the number of points
-			g.setFont(new Font("Corsiva Hebrew", Font.PLAIN, 40));
-			g.setColor(Color.RED);
-			String str1 = "Points: " + eaten + "!";
-			g.drawString(str1, 20, 35);
-			
-			//	displays the health bar
+			// displays the score bar
+			g.setFont(new Font("Corsiva Hebrew", Font.PLAIN, 20));
+						g.setColor(Color.GREEN);
+						g.drawRect(20, 70, 350, 20);
+						g.fillRect(20, 70, (int)(350 * eaten/50), 20);
 			
+			// displays the number of points
+			g.setFont(new Font("Corsiva Hebrew", Font.PLAIN, 20));
+			g.setColor(Color.WHITE);
+			String str1 = "Points: " + eaten + "/50";
+			g.drawString(str1, 135, 87);
+
 
+			// display the health bar
+			
 			g.setFont(new Font("Corsiva Hebrew", Font.PLAIN, 20));
 			g.setColor(Color.RED);
-			g.drawString("Health", 30, 90);
 			g.drawRect(20, 95, 350, 20);
 			g.fillRect(20, 95, (int)(350 * health/maxHealth), 20);
 
+			// display the points of health inside the health bar
+			g.setFont(new Font("Corsiva Hebrew", Font.PLAIN, 20));
+						g.setColor(Color.WHITE);
+			String heal = "My health: " + health + "/50";
```

Comment:
Oops.  One more.  I think I'd said this earlier, but it got lost in an edit.  This should be `"Health: "` to match `"Points: "`.